DataTool/ToolLogic/Render/RenderUIElements.cs

```diff
@@ -19,7 +19,9 @@ public class teResourceGUIDSerializer : IDragonMLSerializer {
 
         public object Print(object instance, Dictionary<object, int> visited, IndentHelperBase indents, string fieldName, DragonMLSettings settings) {
             var hmlNameTag = fieldName == null ? "" : $" hml:name=\"{fieldName}\"";
-
+            //needed for xml namespace support
+            settings.Namespaces.Add("tank", "https://yretenai.com/dragonml/v1");
```

Comment:
This will crash if teResourceGUIDSerializer gets called twice.

KeyAlreadyAddedException.

Use `settings.Namespaces["tank"] = `